src/shared/components/palette/datetime-picker/datetime-picker.component.ts

```diff
@@ -0,0 +1,157 @@
+import {
+  NGX_MAT_DATE_FORMATS,
+  NgxMatDateAdapter,
+  NgxMatDateFormats,
+  NgxMatDatetimePicker
+} from '@angular-material-components/datetime-picker';
+import { Component, ElementRef, Inject, Input, OnInit, ViewChild, ViewEncapsulation } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { MAT_DATE_LOCALE, ThemePalette } from '@angular/material';
+import { AbstractFormFieldComponent } from '../base-field/abstract-form-field.component';
+import { CaseField } from '../../../domain';
+import { NgxMatMomentAdapter, NGX_MAT_MOMENT_DATE_ADAPTER_OPTIONS } from '@angular-material-components/moment-adapter';
+import { CUSTOM_MOMENT_FORMATS } from './datetime-picker-utils';
+import { FormatTranslatorService } from '../../../services/case-fields/format-translator.service';
+import { Moment } from 'moment/moment';
+import moment = require('moment/moment');
+
+@Component({
+  selector: 'ccd-datetime-picker',
+  templateUrl: './datetime-picker.component.html',
+  styleUrls: ['./datetime-picker.component.scss'],
+  encapsulation: ViewEncapsulation.None,
+  providers: [
+    {provide: NGX_MAT_DATE_FORMATS, useValue: CUSTOM_MOMENT_FORMATS},
+    {provide: NgxMatDateAdapter, useClass: NgxMatMomentAdapter},
+    {provide: NGX_MAT_MOMENT_DATE_ADAPTER_OPTIONS, useValue: {useUtc: true}}]
+})
+
+export class DatetimePickerComponent extends AbstractFormFieldComponent implements OnInit {
+
+  public disabled = false;
```

Comment:
I'll remove this because there was an expectation originally that the datetime picker might be disabled in some circumstances but now this is when the write date component is used

Reply:
Removed